tests/console/yast2_vnc.pm

```diff
@@ -36,31 +36,37 @@ sub run {
 
     # check Remote Administration VNC got started
     assert_screen 'yast2_vnc_remote_administration';
-    # enable remote administration
-    send_key 'alt-a';
-    if (is_sle('<15') || is_leap('<15.0')) {
-        # open port in firewall if it is eanbaled and check network interfaces, check long text by send key right.
-        if (check_screen 'yast2_vnc_open_port_firewall', 30) {
-            send_key 'alt-p';
-            send_key 'alt-d';
-            assert_screen 'yast2_vnc_firewall_port_details';
-            send_key 'alt-e';
-            for (1 .. 5) { send_key 'right'; }
-            send_key 'alt-a';
-            send_key 'alt-o';
-        }
-        send_key 'alt-o';    # ok
+    if (check_var('ARCH', 's390x')) {
+        send_key $cmd{ok};
```

Comment:
Seeing the logic below, I believe this may differ on SLE 12 and SLE 15. Also we got 4 levels of nesting, maybe we could extract it to separate method like `open_firewall_port` and then it would be clear just by the code, that we skip it for s390x. Last step of pressing ok/next will be the same, I guess.